Automattic/jetpack
# Blocks: remove custom spinner styles

## Proposed changes:

This should ensure that the Spinner doesn't conflict with other spinners in the editor.

It was added in ed5175dcfa9a5298b55e499df0a7bcbaff0dcb9a, as part of #16302, but removing it doesn't seem to break the spinner's look.

### Other information:


## Jetpack product discussion

References:

* https://github.com/WordPress/gutenberg/pull/49695
* https://github.com/WordPress/gutenberg/pull/37551
* p1683530650412459-slack-C0299DMPG

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Check out this branch
* Go to Posts > Add New
* Add a Story block
* As you interact with the block, check that the loading spinner that appears is displayed properly.
* It should look good on the frontend as well

Here is an example post:
https://crucial-fennec.jurassic.ninja/2023/05/08/a-story/
